treq/client.py

```diff
@@ -205,7 +206,10 @@ def request(self, method, url, **kwargs):
         wrapped_agent = CookieAgent(self._agent, cookies)
 
         if kwargs.get('allow_redirects', True):
-            wrapped_agent = RedirectAgent(wrapped_agent)
+            if kwargs.get("browser_like_redirects", False):
```

Comment:
I don't think anything enforces it but it looks like every other string in this file uses `'` instead of `"`.